lib/settings/pages/comment_appearance_settings_page.dart

```diff
@@ -309,6 +317,18 @@ class _CommentAppearanceSettingsPageState extends State<CommentAppearanceSetting
               ),
             ),
           ),
+          SliverToBoxAdapter(
+            child: Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 16.0),
+              child: ToggleOption(
+                description: LocalSettings.commentShowUserInstance.label,
```

Comment:
Just one last thing and I think we're good! Can we change this to `l10n.commentShowUserInstance` (and add the entry to app_en.arb) rather than use `LocalSettings.commentShowUserInstance.label` since we want it to be localized to the proper strings?

Reply:
Resolved